package/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:1.13 AS helm
-RUN git -C / clone --branch v3.3.3-rancher3 --depth=1 https://github.com/rancher/helm
+FROM golang:1.16 AS helm
+RUN git -C / clone --branch v3.5.4-rancher.1 --depth=1 https://github.com/rancher/helm
```

Comment:
Has the Helm `3.5.2` situation been solved? (or is it even relevant to this repo?)

Reply:
I thought it was solved for 2.6?

Reply:
Has not merged yet

Reply:
that's probably going to break all catalog apps....